src/style/index.scss

```
@@ -601,6 +601,12 @@ textarea.invalid {
     margin: 0 auto;
 }
 
+.frame-text-spaced {
+    @extend .frame-text;
```

Stylelint (17.3.0)

[error] Expected a placeholder selector (e.g. %placeholder) to be used in `@extend` (scss/at-extend-no-missing-placeholder)

(scss/at-extend-no-missing-placeholder)